app/models/fe/page.rb

```diff
@@ -53,7 +53,11 @@ class Page < ActiveRecord::Base
     # end
 
     def has_questions?
-      questions.present? || question_grids.present? || question_grid_with_totals.present?
+      all_questions.present?
+    end
+
+    def all_questions
+      all_elements.where("kind NOT IN('Fe::Paragraph', 'Fe::Section', 'Fe::QuestionGrid', 'Fe::QuestionGridWithTotal')")
```

Comment:
I agree, I had put it as a scope here: https://github.com/CruGlobal/qe/blob/elements-db-cache/app/models/fe/element.rb#L26  Scope seemed to make the most sense, but I can put it as a constant if you prefer.

I just forgot to change this one over.. should read just "all_elements.questions"